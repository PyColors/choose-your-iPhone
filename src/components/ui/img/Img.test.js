import React from 'react';
test('Render a img', () => {
    const wrapper = shallow(
        <figure>Hello Jest!</figure>
    );
    expect(wrapper).toMatchSnapshot();
});
