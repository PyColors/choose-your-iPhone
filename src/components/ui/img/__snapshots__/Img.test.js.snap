// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render a img 1`] = `
<figure>
  Hello Jest!
</figure>
`;
